tests/Integration/DataFixtures/ORM/SettlementRepositoryTest/test_it_finds_all_available_periods.yaml

```diff
@@ -88,8 +88,8 @@ BitBag\OpenMarketplace\Component\Settlement\Entity\Settlement:
     vendor: '@vendor_bruce'
     total_amount: 100500
     total_commission_amount: 1029
-    start_date: '<date_create("first day of last month")>'
-    end_date: '<date_create("last day of last month")>'
```

Comment:
Why? It'll give a different result depending on the month, and it might affect the tests if they're based on that (idk whether it occurs)

Reply:
oh, forgot to change it in the second place. Will do right now.

Reply:
(discard my initial reply)

Exactly! This depends on the current month, and I don't think it's a good idea. That's why I'm changing it to June specifically.

Reply:
Ah, ok. I guess it'd be better to use some kind of mocked calendar for that (so we say, hey, it's June today!) but we can leave it for now as it works. Probably we'll touch while being refactored later;